art/estimators/object_detection/pytorch_object_detector.py

```diff
@@ -263,25 +298,32 @@ def _get_losses(
         """
         self._model.train()
 
+        self.set_dropout(False)
+        self.set_multihead_attention(False)
```

Comment:
Yes, these are required for `PyTorchDetectionTransformer` since it is a subclass of `PyTorchObjectDetector`. Since `PyTorchObjectDetector` and `PyTorchYolo` do not have attention layers, this is just a no-op and does not affect functionality.